src/main/java/net/sf/jabref/logic/util/io/FileUtil.java

```diff
@@ -111,46 +110,38 @@
     /**
      * Copies a file.
      *
-     * @param source         File Source file
-     * @param dest           File Destination file
-     * @param deleteIfExists boolean Determines whether the copy goes on even if the file
-     *                       exists.
-     * @return boolean Whether the copy succeeded, or was stopped due to the
-     * file already existing.
+     * @param pathToSourceFile File Source file
```

Comment:
The parameters are now `Path` objects directing to the file rather than `File` objects.